src/Directory.Build.props

```diff
@@ -37,6 +37,11 @@
         CS1591: Missing XML comment for publicly visible type or member 'Type_or_Member'
     -->
     <NoWarn>$(NoWarn);CS1591</NoWarn>
+
+    <!-- Add more shared configuration properties -->
+    <PropertyGroup>
+        <NetCoreVersion>6.0</NetCoreVersion>
+    </PropertyGroup>
```

Comment:
I don't think this needs to be in it's own property group, and is causing the build to fail